lighthouse-core/test/lib/drivers/driver-test.js

```diff
@@ -80,4 +82,23 @@ describe('Browser Driver', () => {
     assert.notEqual(opts.url, req1.url, 'opts.url changed after the redirects');
     assert.equal(opts.url, req3.url, 'opts.url matches the last redirect');
   });
+
+  it('will clear origin of a page', () => {
```

Comment:
is this something we can really test? Pretty much all of it is up to the browser to get right, so I'm not sure if there's any value in testing it